File: javascript/1190.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var valores = input.split('\n'), count = 0, sum = 0;
var oper = String(valores.shift()).trim();

for (let i = 0; i < 12; i++) {
    for (let j = 0; j < 12; j++) {

        var val = parseFloat(valores.shift());

        if ((j > 6 && i < 6 && j > 11 - i) || (i > 5 && j > i)) {
            sum += val;
            count++;
        }
    }
}
console.log(oper == 'S' ? sum.toFixed(1) : (sum / count).toFixed(1));

File: C/1190.c
#include <stdio.h>

int main() {
    int count = 0, i, j;
    float sum = 0, val;
    char oper;
    
    scanf("%c", &oper);
    
    for (i = 0; i < 12; i++) {
        for (j = 0; j < 12; j++) {
            scanf("%f", &val);
            if ((j > 6 && i < 6 && j > 11 - i) || (i > 5 && j > i)) {
                sum += val;
                count++;
            }
        }
    }
    printf("%.1f\n", oper == 'S' ? sum : sum / count);
}